README.md

```diff
@@ -1,78 +1,38 @@
-[![Actions Status](https://github.com/jrebecchi/GraphQL-Auth-Service/workflows/Node%20CI/badge.svg)](https://github.com/JohannC/GraphQL-Auth-Service/actions)
-[![Coverage Status](https://coveralls.io/repos/github/jrebecchi/GraphQL-Auth-Service/badge.svg?branch=master)](https://coveralls.io/github/jrebecchi/GraphQL-Auth-Service?branch=master)
-[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/jrebecchi/GraphQL-Auth-Service-Heroku)
+<p align="center">
+<img src="/docs/_static/banner.svg">
+<i>Express authentication middleware, using GraphQL and JSON Web Tokens.</i><br/><br/>
+<a href="https://coveralls.io/github/jrebecchi/GraphQL-Auth-Service?branch=master">
+  <img src="https://coveralls.io/repos/github/jrebecchi/GraphQL-Auth-Service/badge.svg?branch=master">
+</a>
+<a href="https://github.com/jrebecchi/GraphQL-Auth-Service/workflows/Node%20CI/badge.svg">
+  <img src="https://img.shields.io/github/workflow/status/jrebecchi/GraphQL-Auth-Service/Node CI?label=tests">
+</a>
+<a href="https://heroku.com/deploy?template=https://github.com/jrebecchi/GraphQL-Auth-Service-Heroku">
+  <img src="https://img.shields.io/badge/heroku-deploy-blueviolet?logo=heroku">
+</a>
+</p>
 
-[![GraphQL Auth Service - Banner](https://nusid.net/img/graphql-auth-service-banner.svg)](#)
+GraphQL Auth Sevice is an authentication middleware for Express handling login, registration, password recovery and account management with GraphQL & JSON Web Tokens.
 
-An authentication middleware for Express handling login, registration, password recovery and account management with GraphQL & JSON Web Tokens. It features [a flexible user data model that you can customize](#extendedschema). [Try the live demo](https://nusid.net/graphql-auth-service). 
+It is a free & open-source alternative to [Firebase Authentication](https://firebase.google.com/products/auth/).
 
-*It is kind of a free & open-source alternative to [Firebase Authentication](https://firebase.google.com/products/auth/).*
+- [**Documentation**](https://jrebecchi.github.io/GraphQL-Auth-Service) — Consult the quick start guide and the online documentation.
 
 ## Features
-  * Registration
-  * Login
-  * Email verification
-  * Password recovery
-  * Account modification
-  * Account deletion
-  * API to fetch public user info
-  * [XSS](https://www.owasp.org/index.php/Cross-site_Scripting_(XSS)) and [CSRF](https://www.owasp.org/index.php/Cross-Site_Request_Forgery_(CSRF)) protection
```

Comment:
- Authentication based on JSONWebTokens
- XSS and CSRF protection
- Shape it with your own User Data Model in 1 sec.
- Easy to integrate into web apps / mobile apps
- Easy to scale
- Test IDE included for development